shared/tasks/components.mjs

```diff
@@ -11,7 +11,7 @@ import { files } from './index.mjs'
  * Generate fixtures.json from component data
  *
  * @param {AssetEntry[0]} pattern - Path to ${componentName}.yaml
- * @param {AssetEntry[1]} options - Asset options
+ * @param {Pick<AssetEntry[1], "srcPath" | "destPath">} options - Asset options
```

Comment:
This task already provides its own `filePath` and `fileContents`

Here we're narrowing down the types to _warn_ when passing in options that aren't picked